### PR TITLE
Feature/low level qc

### PR DESCRIPTION
**Pull request by sb43**
Added code to generate low level QC plot
 Runs mageck on normalised and corrected fold changes

### CODE REVIEW

File: .pytest_cache/v/cache/lastfailed [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (keiranmraine) [inline .pytest_cache/v/cache/lastfailed]: This should not be committed

File: .pytest_cache/v/cache/nodeids [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[
```
Comment (keiranmraine) [inline .pytest_cache/v/cache/nodeids]: Should not be committed (all of this dir tree)

File: pyCRISPRcleanR.egg-info/PKG-INFO [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Metadata-Version: 1.2
```
Comment (keiranmraine) [inline pyCRISPRcleanR.egg-info/PKG-INFO]: `*.egg-info` folder should not be committed

**Review by keiranmraine — DISMISSED** [review]
There is no licensing at all on this project, please follow the docs on the following link to add at least the basic AGPL3 licence:

https://help.github.com/articles/adding-a-license-to-a-repository/

**Comment by sb43** [comment]
Hi Keiran, Dave,
Please review, I have added License and few additional plots.
Thanks,
Shriram

File: .pytest_cache/v/cache/lastfailed (hunk 1 above)
Comment (sb43) [reply .pytest_cache/v/cache/lastfailed]: added to .ignore file

**Review by sb43 — COMMENTED** [review]
reviewed all requested changes

**Review by mcast — APPROVED** [review]
This review is for 8ee5e6dc..eef65b0 following @keiranmraine's.  I see his requested changes are fixed.  I haven't tried to install or run it.

I have some suggestions which you might consider, but they are not change requests

* Where the case of the project name is significant, it might be worth examining the outliers:

```
$ git grep -ni pyCRISPRcleanR | perl -pe 's{[^a-zA-Z]+}{\n}g' | grep -i pyCRISPRcleanR | sort | uniq -c
      1 pycrisprcleanr
     52 pyCRISPRcleanR
      6 pyCRISPRCleanR
```

* Some Docstrings list the parameters and return information in what looks like a stub format.  Where there is no extra explanation of the meanings of these, what's the value of the doc?  (I don't know where it shows up, in any particular doc tool)
    * I found several styles when I used `git grep -10 '"""'`
* Some `print`s or commented `print`s which seem to be leftover debug statements are removed, but there are others left over.  I listed them with `git grep print` (only 26 lines out)
    * Possibly some of these could be removed or replaced with `log.info` calls?
* Functionality added without tests being added.  Is that OK for this project?
* Hardwired strings `'./myfile'`, `'/raw_counts_plotly'` might be worth reconsidering?
  * It may be helpful to document them for their meaning
  * Particularly when they are re-used several times, take them from a "constant"?
* Where `sys.exit` is called, it may reduce support requests if the text would make it clearer that
  1. this is an error and the run is aborted
  2. what to do instead

Hope these are useful, m

**Comment by mcast** [comment]
I think that's everything - over to @sb43

File: README.md [hunk 4]
```diff
@@ -93,6 +93,49 @@ Various exceptions can occur for malformed input files.
  * <treatment sample fold chages: fold changes 1..N> (postfixed _cf)
  * avgFC_cf: average fold change values based on corrected counts
 
+ List of plotly(html) and pdf plots generated for given analysis run
```
Comment (mcast) [inline README.md]: (Inspired by my current task of figuring out the meaning and origin of files which have lost their metadata,) will these output files include any metadata about the inputs or flags?  Either machine readable or human readable.
I don't know whether it's worth the effort to add it...  it seems potentially useful but it's not part of this feature branch, and maybe not a requested feature at all.  Do you want a wishlist issue for it?

File: README.md [hunk 5]
```diff
@@ -111,30 +113,30 @@ Various exceptions can occur for malformed input files.
  * 3_fold_changes_histogram.html
  * 3_fold_changes_correlation_matrix.html
 
- 4. stats plots: precision recall and ROC curves based on known tru positive gene set
+ 4. stats plots: precision recall and ROC curves based on known tru positive sgRNA/gene set
+    [generated only when ```----gene_signatures``` flag is set]
```
Comment (mcast) [inline README.md]: ```----gene_signatures```, really four dashes?  Seems unusual.

**Pull request merged**